test/lib/cipherlinks.test.js

```diff
@@ -1,38 +1,13 @@
 const test = require('tape')
 const { isMsg, isFeedId } = require('ssb-ref')
-const { FeedId, MsgId } = require('../../lib/cipherlinks')
+const { FeedId, MsgId } = require('../helpers/cipherlinks')
 
 test('Cipherlink/FeedId', t => {
-  const feedId = '@YXkE3TikkY4GFMX3lzXUllRkNTbj5E+604AkaO1xbz8=.ed25519'
-  const id = new FeedId(feedId)
-
-  const expected = Buffer.concat([
-    Buffer.from([0]), // type = feed
-    Buffer.from([0]), // format = ssb/classic
-    Buffer.from('YXkE3TikkY4GFMX3lzXUllRkNTbj5E+604AkaO1xbz8=', 'base64')
-  ])
-
-  t.deepEqual(id.toTFK(), expected, 'toTFK')
-  t.deepEqual(id.toSSB(), feedId, 'toSSB')
-
```

Comment:
I assume this is now all tested over in `ssb-bfe`?

Reply:
yes